From 80d2caebaf360db20c0a0f9bf6245f8e92845ff2 Mon Sep 17 00:00:00 2001
From: Michael Ray <michael.ray@arm.com>
Date: Tue, 15 Sep 2020 11:56:34 -0500
Subject: [PATCH] Update create-dev-identity.sh path location for snap

Intended to be patched against snap-pelion-edge pe-utils install
---
 identity-tools/generate-identity.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/identity-tools/generate-identity.sh b/identity-tools/generate-identity.sh
index 2453f6e..9fc5e61 100755
--- a/identity-tools/generate-identity.sh
+++ b/identity-tools/generate-identity.sh
@@ -58,7 +58,7 @@ execute () {
                 cp ${IDENTITY_DIR}/identity.json ${IDENTITY_DIR}/identity_original.json
             fi
             IFS='.' read -ra ADDR <<< "$lwm2mserveruri"
-            ./developer_identity/create-dev-identity.sh\
+            $SNAP/wigwag/pe-utils/identity-tools/developer_identity/create-dev-identity.sh\
                 -d \
                 -m ${ADDR[2]}\
                 -p DEV0\
-- 
2.10.1.windows.1

